### PR TITLE
fix(api): make access internal to self-managed bridge

### PR DESCRIPTION
**Pull request by tatarco**
### What changed? Why was the change needed?
Issue with preview endpoint taking too long, make the access internal

### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts [hunk 1]
```diff
@@ -270,13 +270,7 @@ export class ExecuteBridgeRequest {
...
 
   private getApiUrl(): string {
-    const apiUrl = process.env.API_ROOT_URL;
-
-    if (!apiUrl) {
-      throw new Error('API_ROOT_URL environment variable is not set');
-    }
-
-    return apiUrl;
+    return `http://localhost:${process.env.PORT}`;
```
Comment (djabarovgeorge) [inline libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts]:
```suggestion
    return `http://127.0.0.1:${process.env.PORT}`;
```

Comment (djabarovgeorge) [inline libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts]: having this code running in `Worker` ECS instance will the `API` ECS instance will be in the same network?

**Review by rifont — APPROVED** [review]
🦾

File: libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts (hunk 1 above)
Comment (rifont) [reply libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts]: @djabarovgeorge could you please elaborate on the reason for this suggestion?

Comment (rifont) [reply libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts]: We opted for an approach to only use the localhost loopback for the preview action, where it's guaranteed to be called from the API today.

Comment (djabarovgeorge) [reply libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts]: can you elaborate please? not sure i follow how it is not an issue for preview.

**Pull request merged**